src/app/core/components/marios-dialog/marios-dialog.component.html

```diff
@@ -0,0 +1,41 @@
+<mat-dialog-content class="mat-typography">
+  <div class="container-fluid dialog-container">
+    <div class="dialog-header">
+      <button mat-icon-button (click)="onCloseDialog()">
+        <img src="assets/images/close.svg" alt="" />
+      </button>
+    </div>
+    <div class="dialog-content">
+      <div class="row">
+        <div class="col-md-6">
+          <div class="marios-card">
+            <p class="marios-card-author">
+              From: {{ data.employee.firstName }} {{ data.employee.lastName }}
+            </p>
+          </div>
+        </div>
+
+        <div class="col-md-6 d-flex justify-content-end reaction-container">
+          <p>
+            <img src="assets/images/star.svg" alt="" />
+            {{ data.marios.reaction | reactionMessage }}
+          </p>
+        </div>
+
+        <div class="col-md-12">
+          <div class="marios-card">
+            <h1>{{ data.marios.title }}</h1>
```

Comment:
h1 tag should be used only for application name because of SEO

`An H1 tag is an HTML heading used to mark up a webpage’s main subject`